docs/en/operations/table_engines/mergetree.md

```diff
@@ -47,7 +47,7 @@ CREATE TABLE [IF NOT EXISTS] [db.]table_name [ON CLUSTER cluster]
 
 For descriptions of request parameters, see the [request description](../../query_language/create.md).
 
-**Query clauses**
+**Query Clauses**
```

Comment:
why these are not real headers?

Reply:
I didn't see any practical sence to express this elements as headers, and I don't see it  now :-), but I'll do.

Reply:
Let's just never use this header-like bold and make them small enough `###`.
If it'll be too noisy in ToC or look weird, we'll just fix that.